tests/datasource/fluent/great_expectations.yml

```diff
@@ -12,7 +12,7 @@ fluent_datasources:
         type: table
         name: with_splitter
         table_name: another_table
-        column_splitter:
+        splitter:
```

Comment:
@billdirks I am assuming that the splitter method "split on whole table" -- if still supported -- will work under this renaming (please confirm/elaborate).  Thanks!

Reply:
We have decided not to make a specific splitter for that since it's the default behavior when no splitter is assigned. 

https://greatexpectationslabs.slack.com/archives/C026YSF2JA3/p1677180613670379?thread_ts=1677102805.974729&cid=C026YSF2JA3

All the splitters should work with the rename.